includes/parserhooks/SetParserFunction.php

```diff
@@ -30,6 +30,11 @@ class SetParserFunction {
 	private $messageFormatter;
 
 	/**
+	 * @var Parser
+	 */
+	private $parser = null;
```

Comment:
Why init to `null` explicitly? The doc is also kinda wrong, since it can be`null`